packages/core/src/components/index.js

```diff
@@ -18,6 +18,9 @@ export * from './Review/Review';
 export * from './SkipNav/SkipNav';
 export * from './Spinner/Spinner';
 export * from './StepList/StepList';
+export * from './StepList/Step';
+export * from './StepList/StepLink';
+export * from './StepList/SubStep';
```

Comment:
Why are we exporting these internal components now?

Reply:
I saw that `VerticalNav` was also exporting `VerticalNavItem`, `VerticalNavItemLabel` so I assumed it was intended to expose these ones too. Happy to revert this change, I'm just not super clear on whats considered internal.

Reply:
I don't know what the right choice is here exactly, but my thinking was that if the component doesn't have a purpose outside of its parent and there's no way to build our own parent from the children anyway (like `<StepList><StepItem /></StepList>`, then there's no point in exposing it or documenting it.

Reply:
And if we expose it, we should document it.

Reply:
That makes sense to me,

> but my thinking was that if the component doesn't have a purpose outside of its parent and there's no way to build our own parent from the children anyway

That makes sense to me. Looking at both `VerticalNav` it looks like we shouldn't be exposing those either but I can take care of that in a different PR